Mathlib/NumberTheory/LegendreSymbol/QuadraticChar/GaussSum.lean

```diff
@@ -99,16 +85,15 @@ theorem quadraticChar_card_card [DecidableEq F] (hF : ringChar F ≠ 2) {F' : Ty
     quadraticChar F (Fintype.card F') =
     quadraticChar F' (quadraticChar F (-1) * Fintype.card F) := by
   let χ := (quadraticChar F).ringHomComp (algebraMap ℤ F')
-  have hχ₁ : χ.IsNontrivial := by
+  have hχ₁ : χ ≠ 1 := by
     obtain ⟨a, ha⟩ := quadraticChar_exists_neg_one hF
     have hu : IsUnit a := by
```

Comment:
This is needed (at least) twice. Add a lemma
```lean
lemma quadraticChar_exists_neg_one' (hF : ringChar F ≠ 2) : ∃ a : Fˣ, quadraticChar F a = -1
```
in `NumberTheory.LegendreSymbol.QuadraticChar.Basic`?